AllReadyApp/Web-App/AllReady/Features/Events/EventByMonthQueryHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using AllReady.Models;
+using AllReady.ViewModels.Event;
+using MediatR;
+
+namespace AllReady.Features.Event
+{
+    public class EventByMonthQueryHandler : IRequestHandler<EventByMonthQuery, IEnumerable<EventViewModel>>
+    {
+        private readonly IAllReadyDataAccess dataAccess;
+
+        public EventByMonthQueryHandler(IAllReadyDataAccess dataAccess)
+        {
```

Comment:
absolutelly no problem. I'll fix the code and push the changes soon. 
Just out of curiosity: what's the advantage of phacing out `IAllreadyDataAccess` and working with `AllReadyContext` directly?

any way, it looks like [IAllReadyDataAccess](https://github.com/HTBox/allReady/blob/master/AllReadyApp/Web-App/AllReady/DataAccess/IAllReadyDataAccess.cs) hasn't still been marked as obsolete, am I looking at the wrong place?